snuba/query/query.py

```diff
@@ -128,3 +129,13 @@ def set_offset(self, offset: int) -> None:
         "use the specific accessor methods instead.")
     def get_body(self) -> Mapping[str, Any]:
         return self.__body
+
+
+@dataclass
+class QueryHints:
+    """
```

Comment:
Could you please add to the comment what the difference between the query and the query hints is? It is important to decide where to put stuff going on.